tidy3d/components/monitor.py

```diff
@@ -628,6 +628,21 @@ class AbstractFieldProjectionMonitor(SurfaceIntegrationMonitor, FreqMonitor):
         "in the far field of the device.",
     )
 
+    interval_space: Tuple[
+        pydantic.PositiveInt, pydantic.PositiveInt, pydantic.PositiveInt
+    ] = pydantic.Field(
+        (1, 1, 1),
+        title="Spatial interval",
```

Comment:
I think we do capital case for titles ("Spatial Interval") but it seems this is inconsistent across the code base so not sure.

Reply:
Ok I've applied this to all the titles in `monitor.py`